lms/djangoapps/grades/signals/handlers.py

```diff
@@ -294,3 +294,24 @@ def _emit_event(kwargs):
                 'event_transaction_type': unicode(GRADES_RESCORE_EVENT_TYPE),
             }
         )
+
+    if root_type == 'edx.grades.problem.score_overridden':
```

Comment:
This string should be a field (there's another instance of this on line 277 you could fix, too).